Update required Node version to 18.20.8 (#11692)
withastro/docs: src/content/docs/en/tutorial/1-setup/1.mdx

### Node.js

> For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed, version `v18.20.8` or `v20.3.0`, `v22.0.0` or higher. (`v19` and `v21` are not supported.)

Reviewer: This sentence structure doesn't work as well here now that there are multiple versions "or higher" for each one of them.

What about something like:

```suggestion
For Astro to run on your system, you will also need to have a compatible version of [**Node.js**](https://nodejs.org/en/) installed. Astro supports **even-numbered** Node.js versions. The current minimum supported versions of each are:  `v18.20.8`, `v20.3.0`, and `v22.0.0`. (`v19` and `v21` are not supported.)
```

(Even if v18 must be *exactly* v.18.20.8, if there's no higher version that we *don't* support, I think the wording is fine for a more readable sentence?)

Alternatively, we go into bullet points (which weren't really needed before when it was simple enough to just say in a sentence):

```suggestion
For Astro to run on your system, you will also need to have a compatible version of [**Node.js**](https://nodejs.org/en/) installed. Astro supports **even-numbered** Node.js versions. The current supported versions are:

 - `v.18.20.8`
 - `v.20.3.0` or higher
 - `v.22.0.0` or higher 
```

To check to see whether you already have a compatible version installed, run the following command in your terminal:

```sh
node -v

// Example output
v18.20.8
```

> If the command returns a version number higher than `v18.20.8` or `v20.3.0` (excluding any `v19` or `v21`), you're good to go!

Reviewer: ```suggestion
If the command returns a version number supported by Astro, you're good to go!
```
Again, I don't think we can say this simply here like we could before. So maybe this is sufficient?
